### PR TITLE
Create reveal words screen.

### LINKED ISSUE #26: Reveal 12 Words
App parity - allow users to reveal their 12 words

- [ ] require PIN to reveal 12 words

- [ ] display 12 words

### PR DESCRIPTION
**Pull request by matthewcarlreetz**
Closes #26

### CODE REVIEW

**Review by tyler-whitman — APPROVED** [review]

File: src/features/moreTab/reveal/RevealWordsScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  )
+
+  return (
+    <SafeAreaBox
+      backgroundColor="primaryBackground"
+      flex={1}
+      paddingHorizontal="lx"
+    >
+      <Box flex={1} />
+      <Text variant="h1">{t('account_setup.passphrase.title')}</Text>
```
Comment (tyler-whitman) [inline src/features/moreTab/reveal/RevealWordsScreen.tsx:46]: Looks like this translation is missing

File: src/features/moreTab/reveal/RevealWordsScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+      <TextTransform
+        marginVertical="l"
+        variant="subtitle"
+        i18nKey="account_setup.passphrase.subtitle"
```
Comment (tyler-whitman) [inline src/features/moreTab/reveal/RevealWordsScreen.tsx:50]: This one is missing too

File: src/features/moreTab/reveal/RevealWordsScreen.tsx (hunk 1 above)
Comment (matthewcarlreetz) [reply src/features/moreTab/reveal/RevealWordsScreen.tsx:46]: oh yeah 🤦 , it's in my other pr. I'll make sure that goes in first.

**Review by allenan — APPROVED** [review]
looks good, pending missing translations

**Pull request merged**